chain/client/src/sync/state.rs

```diff
@@ -194,10 +217,14 @@ impl StateSync {
             let mut this_done = false;
             match &shard_sync_download.status {
                 ShardSyncStatus::StateDownloadHeader => {
+                    // StateDownloadHeader is the first step. We want to fetch the basic information about the state (its size, hash etc).
+                    // FIXME: why are we looking only on the first index? - ah right, we assume taht download header has only 1.
```

Comment:
Not clear what this FIXME requires to fix

Reply:
removed.